coa/pkg/apis/v1alpha2/providers/pubsub/redis/redis_test.go

```diff
@@ -297,7 +297,7 @@ func TestRedisPubSubProviderConfigFromMap(t *testing.T) {
 		"numberOfWorkers":   "1",
 		"queueDepth":        "10",
 		"consumerID":        "test-consumer",
-		"processingTimeout": "10",
+		"processingTimeout": "10s",
```

Comment:
Add another test for time format.

Case 1: processingTimeout: 10, redeliverInterval: 10
Case 2: ProcessingTimeout: 10s, redeliverInterval: 10s,

Reply:
Fixed